src/App.tsx

```diff
@@ -21,6 +62,12 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={selectedValue}
+            onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
+              setActivePage(1);
+
+              return setSelectedValue(e.currentTarget.value);
+            }}
```

Comment:
Pls move the function inside `onChange` handler to a separate const and use as a callback here